app/angular/src/client/preview/angular-beta/RendererService.test.ts

```diff
@@ -1,241 +0,0 @@
-import { Component, ɵresetJitOptions } from '@angular/core';
-import { platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
-import { Parameters } from '../types-6-0';
-import { RendererService } from './RendererService';
-
-jest.mock('@angular/platform-browser-dynamic');
-
-declare const document: Document;
-describe('RendererService', () => {
-  let rendererService: RendererService;
-
-  beforeEach(async () => {
-    document.body.innerHTML = '<div id="root"></div>';
-    (platformBrowserDynamic as any).mockImplementation(platformBrowserDynamicTesting);
-    rendererService = new RendererService();
-  });
-
-  afterEach(() => {
-    jest.clearAllMocks();
-
-    // Necessary to avoid this error "Provided value for `preserveWhitespaces` can not be changed once it has been set." :
-    // Source: https://github.com/angular/angular/commit/e342ffd855ffeb8af7067b42307ffa320d82177e#diff-92b125e532cc22977b46a91f068d6d7ea81fd61b772842a4a0212f1cfd875be6R28
-    ɵresetJitOptions();
-  });
-
-  it('should initialize', () => {
-    expect(rendererService).toBeDefined();
-  });
-
-  describe('render', () => {
-    it('should add storybook-wrapper for story template', async () => {
-      await rendererService.render({
-        storyFnAngular: {
-          template: '🦊',
-          props: {},
-        },
-        forced: false,
-        parameters: {},
-      });
-
-      expect(document.body.getElementsByTagName('storybook-wrapper')[0].innerHTML).toBe('🦊');
-    });
-
-    it('should add storybook-wrapper for story component', async () => {
-      @Component({ selector: 'foo', template: '🦊' })
-      class FooComponent {}
-
-      await rendererService.render({
-        storyFnAngular: {
-          props: {},
-        },
-        forced: false,
-        component: FooComponent,
-        parameters: {},
-      });
-
-      expect(document.body.getElementsByTagName('storybook-wrapper')[0].innerHTML).toBe(
-        '<foo>🦊</foo><!--container-->'
-      );
-    });
-
-    it('should handle circular reference in moduleMetadata', async () => {
```

Comment:
This test should probably be moved to `RendererFactory.test.ts`.

Reply:
Yes ! Ty

Reply:
Done